notebooks/random_forest_clean.ipynb

```diff
@@ -0,0 +1,1507 @@
+{
```

Comment:
<p>if you like the below visualization/table better, just delete this cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/srensi/tmprss2/pull/20/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>